person.rb

```diff
@@ -0,0 +1,29 @@
+require 'securerandaom'
+
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unkown', parent_permission: true)
+    @id = SecureRandom.uuid
+    @name = name
+    @parent_permission = parent_permission
+    @age = age
+  end
+
+  # rubocop:disable  Naming/PredicateName
+  def is_of_age?
+    return true if @age >= 18
+
+    false
+  end
+  # rubocop:enable Naming/PredicateName
+
+  def can_use_services
+    return true if is_of_age? || @parent_permission
+
+    false
+  end
+
+  private :is_of_age?
```

Comment:
Hi @Omar-Muhamad , Thanks for your review. For this specific suggestion, I used `rubocop:disable` because the linter was asking me to rename the `is_of_age` function however in the project requirement `is_of_age` is the name specified. Should i go ahead and rename it like suggested by the linter of maintain the project requirement and the rubocop:disable

Reply:
Yes. It's ok to rename it like rubocop suggested.